tenets/codelingo/go/avoid-delayed-must-calls/codelingo.yaml

```diff
@@ -20,8 +20,8 @@ tenets:
         @review comment
         go.assign_stmt(depth = any):
           go.rhs:
-            go.call_expr:
-              go.selector_expr:
+            go.call_expr(depth = any):
+              go.selector_expr(depth = any):
```

Comment:
I think we can drop the depth = any from the selector expr and also remove everything above call_expr. I can see the assign_stmt preventing capture of things like this:
```
if !regexp.MustCompile("foo").MatchString("bar") {
	fmt.Println("did not match")
}
```

as the result is never assigned to anything.